doc/specs/vulkan/appendices/VK_EXT_debug_report.txt

```diff
@@ -51,9 +51,9 @@ to the application for events of interest to the application.
 
 === Examples
 
-+VK_EXT_debug_report+ allows an application to register multiple callbacks
+`VK_EXT_debug_report` allows an application to register multiple callbacks
 with the validation layers.
-Some callbacks may log the information to a file, others may cause a debug
+Some callbacks can: log the information to a file, others can: cause a debug
```

Comment:
Why can:? I deliberately chose 'may' because this section describes examples and isn't specifying extension behavior.

Reply:
Because it is describing "something that the Application can achieve", **not** "something that is optional to the Implementation" as per 1.3. Terminology chapter.